doc/fastfetch.1.in

```diff
@@ -0,0 +1,134 @@
+.TH FASTFETCH 1 "@FASTFETCH_BUILD_DATE@" "@CMAKE_PROJECT_NAME@ @CMAKE_PROJECT_VERSION@"
+
+.SH NAME
+fastfetch - a neofetch-like tool for fetching system information and
+displaying them in a pretty way
+
+.SH SYNOPSIS
+
+.B fastfetch
+.I [options...]
+
+.SH DESCRIPTION
+
+Fastfetch is a neofetch-like tool for fetching system information and
+displaying them in a pretty way. It is written mainly in C, with performance
+and customizability in mind.
+Currently, Linux, Android, FreeBSD, MacOS and Windows 7+ are supported.
+
+.SH "EXIT STATUS"
+
+On successfull execution, fastfetch returns zero. If any error happened,
+the exit code will be non-zero.
+
+.SH OPTIONS
+
+.SS "Informative Options"
+.TP
+
+.B \-\-help \fI [command]
```

Comment:
`-h`

Reply:
fixed in 310b3cc071dcd06e0feb6f01f977a8235cd10686